meson.build

```diff
@@ -1960,7 +1960,7 @@ foreach tuple : [['myhostname', 'ENABLE_NSS_MYHOSTNAME'],
                 nss = shared_library(
                         'nss_' + module,
                         sources,
-                        disable_mempool_c,
+                        constructor_c,
```

Comment:
So nss-systemd calls log_parse_environment() already from `setup_logging_once`. I think that should be removed, it doesn't make sense to the setup twice.